app/models/label/LabelTable.scala

```diff
@@ -728,9 +729,21 @@ object LabelTable {
     // Init random function.
     val rand = SimpleFunction.nullary[Double]("random")
 
+    // Prioritize labels to show by filtering out those that have been
+    // validated as "disagree" 3 or more times and have twice as many
+    // disagrees as agrees
+    val prioritizedLabels = Q.queryNA[(Int)](
+      """SELECT label.label_id
+        |FROM "sidewalk".label
+        |INNER JOIN "sidewalk".label_validation ON label.label_id = label_validation.label_id
+        |GROUP BY label.label_id
+        |HAVING COUNT(CASE WHEN validation_result = 2 THEN 1 END) < 3 
```

Comment:
The number of labels we want to include is going to be a lot larger than the number of labels that we want to exclude. And given that we're doing the thing where we run a static query, put the result in a Scala list, then use the `inSet` filtering, it would make more sense to me (and be a bit more performant) to instead query for the labels that we want to exclude. This basically means turning `disagree < 3 OR disagree < 2 * agree` to `disagree > 2 AND disagree >= 2 * agree`.

Reply:
agreed